src/MOI_wrapper.jl

```diff
@@ -152,7 +153,7 @@ end
 
 
 function MOI.optimize!(model::Optimizer)
-    MOI.optimize!(model.optimizer)
+    solution = MOI.optimize!(model.optimizer)
```

Comment:
`optimize!` returns `nothing`

Reply:
for SCS.jl, it returns `MOISolution`. can query it to get slack and dual solution easily

Reply:
added this only for SCS.jl. indeed other solvers dont return anything